packages/cli/src/util/plugins.ts

```diff
@@ -0,0 +1,91 @@
+import code from '../util/output/code';
+import { getColorForPkgName } from '../util/output/color-name-cache';
+import cliPkgJson from '../util/pkg';
+import { scanParentDirs } from '@vercel/build-utils';
+
+const VERCEL_PLUGIN_PREFIX = 'vercel-plugin-';
+
+export async function loadCliPlugins(
+  cwd: string,
+  logError: (errorMessage: string) => void = console.error
```

Comment:
This should probably accept the `Output` object instead of a function

Reply:
Oh, I missed that it was even a thing in the server. I'll update.